sunpy/timeseries/timeseries_factory.py

```diff
@@ -524,7 +523,7 @@ def _check_registered_widgets(self, **kwargs):
         """
 
         WidgetType = self._get_matching_widget(**kwargs)
-
+        source = kwargs.pop('source', None)
```

Comment:
Where did this come from?

Reply:
Again, this needs popping to avoid unwanted kwargs being handed on to another function later.